website/content/docs/connect/manage-traffic/route-to-local-upstreams.mdx

```diff
@@ -83,23 +83,252 @@ You can configure the default routing behavior for all proxies in the mesh as we
+
+```json
+{
+  "@type": "type.googleapis.com/envoy.config.endpoint.v3.ClusterLoadAssignment",
+  "cluster_name": "web.default.dc1.internal.161d7b5a-bb5f-379c-7d5a-1fc7504f95da.consul",
+  "endpoints": [
+    {
+      "lb_endpoints": [
+        {
+          "endpoint": {
+            "address": {
+              "socket_address": {
+                "address": "10.42.2.6",
+                "port_value": 20000
+              }
+            },
+            "health_check_config": {}
+          },
+          ...
+        },
+        ...
+      ],
+      "locality": {}
+    },
+    {
+      "lb_endpoints": [
+        {
+          "endpoint": {
+            "address": {
+              "socket_address": {
+                "address": "10.42.3.6",
+                "port_value": 20000
+              }
+            },
+            "health_check_config": {}
+          },
+          ...
+        },
+        ...
+      ],
+      "locality": {},
+      "priority": 1
+    },
+    {
+      "lb_endpoints": [
+        {
+          "endpoint": {
+            "address": {
+              "socket_address": {
+                "address": "10.42.0.6",
+                "port_value": 20000
+              }
+            },
+            "health_check_config": {}
+          },
+          ...
+        },
+        ...
+      ],
+      "locality": {},
+      "priority": 2
+    }
+  ],
+  ...
+}
+```
+
+To force a failover for testing purposes, scale the upstream service instances in the downstream's local zone (or region, if no local zone instances are avaiable) to zero.
+
+<Tip>
+Keep in mind the following:
+  - Priority for failover is in ascending order, starting with `0`. Priority `0` (the "highest" priority) will not be explicitly visible in xDS output, since it is the default value for that field.
+  - Once Envoy failover configuration is in place, the specific timing of failover is determined by the downstream Envoy proxy, not Consul. Consul health status may not correspond 1-1 with Envoy's failover behavior, which is also dependent on outlier detection.
+</Tip>
```

Comment:
All of this is a new topic and needs to be placed into its own page. 

```suggestion
## Observing routing changes

This feature relies on configuring Envoy's built-in [`overprovisioning_factor`](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/endpoint/v3/endpoint.proto#config-endpoint-v3-clusterloadassignment) and [outlier detection](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/outlier_detection.proto#config-cluster-v3-outlierdetection) configuration to enforce failover behavior. Because Envoy does not provide granular metrics specific to failover or endpoint traffic within a cluster, the effects of enabling this feature are best observed using external observability tools that expose network traffic within your environment. _The immediate effect of enabling the feature should be the local routing of traffic from each downstream service instance to the most-local set of healthy upstream instances available._

To verify locality-aware routing and failover configuration is in place, you can inspect Envoy's xDS config dump for a downstream proxy. Refer to the [consul-k8s CLI docs](https://developer.hashicorp.com/consul/docs/k8s/k8s-cli#proxy-read) for details on how to obtain this on Kubernetes; for other workloads, use the Envoy [admin interface](https://www.envoyproxy.io/docs/envoy/latest/operations/admin) and ensure that you [include EDS](https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--config_dump?include_eds). Inspect the [priority](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/priority#arch-overview-load-balancing-priority-levels) on each set of endpoints under the upstream `ClusterLoadAssignment` in the `EndpointsConfigDump`. Alternatively, the same priorities should be visibile within the output of the [`/clusters?format=json`](https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--clusters?format=json) admin endpoint.

```json
{
  "@type": "type.googleapis.com/envoy.config.endpoint.v3.ClusterLoadAssignment",
  "cluster_name": "web.default.dc1.internal.161d7b5a-bb5f-379c-7d5a-1fc7504f95da.consul",
  "endpoints": [
    {
      "lb_endpoints": [
        {
          "endpoint": {
            "address": {
              "socket_address": {
                "address": "10.42.2.6",
                "port_value": 20000
              }
            },
            "health_check_config": {}
          },
          ...
        },
        ...
      ],
      "locality": {}
    },
    {
      "lb_endpoints": [
        {
          "endpoint": {
            "address": {
              "socket_address": {
                "address": "10.42.3.6",
                "port_value": 20000
              }
            },
            "health_check_config": {}
          },
          ...
        },
        ...
      ],
      "locality": {},
      "priority": 1
    },
    {
      "lb_endpoints": [
        {
          "endpoint": {
            "address": {
              "socket_address": {
                "address": "10.42.0.6",
                "port_value": 20000
              }
            },
            "health_check_config": {}
          },
          ...
        },
        ...
      ],
      "locality": {},
      "priority": 2
    }
  ],
  ...
}
```

To force a failover for testing purposes, scale the upstream service instances in the downstream's local zone (or region, if no local zone instances are avaiable) to zero.

<Tip>
Keep in mind the following:
  - Priority for failover is in ascending order, starting with `0`. Priority `0` (the "highest" priority) will not be explicitly visible in xDS output, since it is the default value for that field.
  - Once Envoy failover configuration is in place, the specific timing of failover is determined by the downstream Envoy proxy, not Consul. Consul health status may not correspond 1-1 with Envoy's failover behavior, which is also dependent on outlier detection.
</Tip>
```

Reply:
Where would we want to put that separate topic? This is highly specific to the feature, so are you suggesting we break it out and link to it just to control content length?

(I can't tell what's being suggested to change because of the nested ``` marks - it looks like maybe there isn't a suggestion for this one and it was primarily a comment above moving.)